lua/lazyvim/plugins/extras/ai/copilot.lua

```diff
@@ -69,42 +83,38 @@ return {
             copilot_cmp._on_insert_enter({})
           end, "copilot")
         end,
+        specs = {
+          {
+            "zbirenbaum/copilot.lua",
+            opts = { suggestion = { enabled = false } },
+          },
```

Comment:
Is there a reason for this, since it's also defined in the `dependencies` on line 75? From what I can understand `specs` just don't load the plugin spec, but it's already listed in `dependencies`. So, I'm just wondering about the reason of this spec existing in both places.

Reply:
The code changed quite a bit in the meantime, so this is probably no longer relevant :)